pkg/genericapiserver/endpoints/handlers/rest.go

```diff
@@ -140,7 +140,7 @@ func GetResource(r rest.Getter, e rest.Exporter, scope RequestScope) restful.Rou
 			options := metav1.GetOptions{}
 			if values := req.Request.URL.Query(); len(values) > 0 {
 				exports := metav1.ExportOptions{}
-				if err := scope.ParameterCodec.DecodeParameters(values, schema.GroupVersion{Version: "v1"}, &exports); err != nil {
+				if err := metainternalversion.ParameterCodec.DecodeParameters(values, scope.MetaGroupVersion, &exports); err != nil {
```

Comment:
again, this `metav1.ExportOptions` is actually the internal type which happens to be equal to metav1. Type aliases would be nice if our conversion handled those without performance hit (does it?).